UnitTests/CheckboxTests.cs

```diff
@@ -327,44 +335,56 @@ public void TextAlignment_Justified ()
 				Height = Dim.Fill (),
 				Title = "Test Demo 你"
 			};
-			win.Add (checkBox);
+			win.Add (checkBox1, checkBox2);
 			Application.Top.Add (win);
 
 			Application.Begin (Application.Top);
-			((FakeDriver)Application.Driver).SetBufferSize (30, 5);
+			((FakeDriver)Application.Driver).SetBufferSize (30, 6);
+
+			Assert.Equal (TextAlignment.Justified, checkBox1.TextAlignment);
+			Assert.Equal (new Rect (1, 1, 25, 1), checkBox1.Frame);
+			Assert.Equal (new Size (25, 1), checkBox1.TextFormatter.Size);
+			Assert.Equal ("Check first out 你", checkBox1.Text);
+			Assert.Equal ("╴ Check first out 你", checkBox1.TextFormatter.Text);
+			Assert.False (checkBox1.AutoSize);
+			Assert.Equal (TextAlignment.Justified, checkBox2.TextAlignment);
+			Assert.Equal (new Rect (1, 2, 25, 1), checkBox2.Frame);
+			Assert.Equal (new Size (25, 1), checkBox2.TextFormatter.Size);
+			Assert.Equal ("Check second out 你", checkBox2.Text);
+			Assert.Equal ("╴ Check second out 你", checkBox2.TextFormatter.Text);
+			Assert.False (checkBox2.AutoSize);
 
-			Assert.Equal (TextAlignment.Justified, checkBox.TextAlignment);
-			Assert.Equal (new Rect (1, 1, 25, 1), checkBox.Frame);
-			Assert.Equal (new Size (25, 1), checkBox.TextFormatter.Size);
-			Assert.Equal ("Check this out 你", checkBox.Text);
-			Assert.Equal ("╴ Check this out 你", checkBox.TextFormatter.Text);
-			Assert.False (checkBox.AutoSize);
 
 			var expected = @"
 ┌ Test Demo 你 ──────────────┐
 │                            │
-│ ╴  Check  this  out  你    │
+│ ╴   Check  first  out  你  │
```

Comment:
I don't understand why the extra space here (before `Check`) makes sense. 

What am i missing???

Reply:
The checkbook has a justify alignment option and that why is displaying this format.

Reply:
Got it. Missed that. Thanks.